plugins/org.eclipse.fordiac.ide.gef/plugin.xml

```diff
@@ -139,30 +161,45 @@
            		</or>                  
          	</activeWhen>
 		</handler>    
+	</extension>
+	<extension
+		point="org.eclipse.ui.contexts">
+		<context
+			id="org.eclipse.fordiac.ide.gef"
+			description="%context.graphicalEditing.description"
+			name="%context.graphicalEditing.name"
+			parentId="org.eclipse.ui.contexts.window">
+		</context>
 	</extension>  
 	<extension
 			point="org.eclipse.ui.bindings">
 		<key
         	commandId="org.eclipse.fordiac.ide.gef.zoom100"
-            contextId="org.eclipse.ui.contexts.window"
+            contextId="org.eclipse.fordiac.ide.gef"
             schemeId="org.eclipse.ui.defaultAcceleratorConfiguration"
-            sequence="M1+0">
-      	</key>
+            sequence="M1+0"/>
 		<key
         	commandId="org.eclipse.fordiac.ide.gef.zoomfitpage"
-            contextId="org.eclipse.ui.contexts.window"
+            contextId="org.eclipse.fordiac.ide.gef"
             schemeId="org.eclipse.ui.defaultAcceleratorConfiguration"
-            sequence="M1+M2+0">
-      	</key>
+            sequence="M1+M2+0"/>
+      	<key
+	        commandId="org.eclipse.fordiac.ide.gef.zoomIn"
+	        contextId="org.eclipse.fordiac.ide.gef"
+			schemeId="org.eclipse.ui.defaultAcceleratorConfiguration"
+	        sequence="M1++"/>
```

Comment:
I was not sure about it, I guess it is more used in english keyboards, added it. Also I switched back to the orginal zoom command id from the text editor. Otherwise the toolbar entries will only be active on graphical editors which felt strange.